PowerControl/Menu/MenuItemWithOptions.cs

```diff
@@ -91,6 +91,11 @@ public void Set(String value, bool immediately = false)
             delayTimer.Enabled = true;
         }
 
+        public void SetValueChanged(Action<MenuItemWithOptions, String?, String> action)
```

Comment:
This is not needed, as someone should use `+=` operator.

Reply:
+= doesn't work if you try to change it outside namespace PowerControl.Menu. But it shouldn't matter if we will move to more general approach